<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"><title>SWISS-MODEL | Workspace | Model Results | 28QjD5</title><style>
			.sequence { font-size: 12px; font-family:"Courier New", Courier, monospace;}
			h2 {font-size: 1.2em; font-family:Arial; margin:20 0 0 0}
			h3 {font-size: 1.0em; font-family:Arial; margin:0 0 0 0}
			
			.borderedTbl>tbody>tr>td, .borderedTbl th{ 
        text-align:center;
  			border-top:1px solid #DDD;
  			border-left:1px solid #DDD;
      }
      .borderedTbl{
        border-bottom:1px solid #DDD;
        border-right:1px solid #DDD; 
      }
      td { 
        -webkit-hyphens: auto;
           -moz-hyphens: auto;
                hyphens: auto; 
              }

			body {
			width:100%; 
			height:100%; 
			background-color: #E7E8EA;
			margin: 0;
			padding: 0;
			word-wrap: break-word;
			font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
			font-size: 10pt;
			}
			#content{
			padding:20px;
			width:780px;
			margin-left:auto;
			margin-right:auto;
			background-color:#FFF;}
			
			#sib_footer {
			background-color: #333333;
			color: #F7F8FA;
			font-family: Verdana;
			font-size: 10px;
			font-weight: normal;
			height: 20px;
			letter-spacing: 1px;
			
			width:760px;
      margin-left:auto;
      margin-right:auto;
      padding:0px 20px 0px 20px
			}
			
			#sib_footer a {
  			color: #FFFFFF;
  			font-family: Verdana;
  			font-size: 10px;
  			font-weight: normal;
  			letter-spacing: 1px;
			}
			a, a:link, a:visited {
			color: #D6161D;
			text-decoration: none;
			} 	

			.alTbl{ font-family:'Courier New', Courier, monospace; }
	   img{ border:0; }
			
	@media print{
	
	#content{
			padding:20px;
			background-color:#FFF;
		}
	body{ 
	    width:100%;
	    height:100%; 
	    background-color: #FFF;
			padding: 0;
		}	
	#sib_footer{
		display:none;
	}
	}
      
    </style></head><body><div id="content"><table border=0 width="100%"><td valign="middle"><a title="SIB Swiss Institute of Bioinformatics" href="https://www.sib.swiss"><img src="images/SIB_logo_20_sm.png"></a></td><td valign="middle"><a title="Biozentrum, University of Basel" href="https://www.biozentrum.unibas.ch"><img src="images/biozentrum_en.png"></a></td><td valign="middle"><div style="font-size:1.9em;">SWISS-MODEL Homology Modelling Report</div></td></tr></table><div style="clear:both"></div><h2>Model Building Report</h2><p>
This document lists the results for the homology modelling project &quot;O00476-2&quot; submitted to SWISS-MODEL workspace
on Dec. 3, 2018, 7:28 p.m..The submitted primary amino acid sequence is given in Table T1.</p><p>If you use any results in your research, please cite the relevant publications:</p><ul><li>Waterhouse, A., Bertoni, M., Bienert, S., Studer, G., Tauriello, G., Gumienny, R.,
      Heer, F.T., de Beer, T.A.P., Rempfer, C., Bordoli, L., Lepore, R., Schwede, T.   
      SWISS-MODEL: homology modelling of protein structures and complexes.
      Nucleic Acids Res. 46(W1), W296-W303 (2018).
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/29788355"
       target="_blank"><img src="images/pubmed.jpg
                                 "
                            width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gky427"
       target="_blank"><img src="images/doi.jpg
                                 "
                            width="30" height="16" align="absmiddle"
                            style="padding-left:2px"></a></li><li><a name="promodII"></a>Guex, N., Peitsch, M.C., Schwede, T. Automated
          comparative protein structure modeling with SWISS-MODEL and
          Swiss-PdbViewer: A historical perspective. Electrophoresis 30,
          S162-S173 (2009).
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/19517507"
       target="_blank"><img src="images/pubmed.jpg
                                 "
                            width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1002/elps.200900140"
       target="_blank"><img src="images/doi.jpg
                                 "
                            width="30" height="16" align="absmiddle"
                            style="padding-left:2px"></a></li><li>Bienert, S., Waterhouse, A., de Beer, T.A.P., Tauriello, G., Studer,
    G., Bordoli, L., Schwede, T. The SWISS-MODEL Repository - new features and
    functionality. Nucleic Acids Res. 45, D313-D319 (2017).
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27899672"
       target="_blank"><img src="images/pubmed.jpg
                                 "
                            width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gkw1132"
       target="_blank"><img src="images/doi.jpg
                                 "
                            width="30" height="16" align="absmiddle"
                            style="padding-left:2px"></a></li><li><a name="qmean"></a>Benkert, P., Biasini, M., Schwede, T. Toward the
        estimation of the absolute quality of individual protein structure
        models. Bioinformatics 27,
        343-350 (2011).
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/21134891"
       target="_blank"><img src="images/pubmed.jpg
                                 "
                            width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/bioinformatics/btq662"
       target="_blank"><img src="images/doi.jpg
                                 "
                            width="30" height="16" align="absmiddle"
                            style="padding-left:2px"></a></li><li><a name="qsqe"></a>Bertoni, M., Kiefer, F., Biasini, M., Bordoli, L.,
        Schwede, T. Modeling protein quaternary structure of homo- and
        hetero-oligomers beyond binary interactions by homology. Scientific
        Reports 7 (2017).
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/28874689"
       target="_blank"><img src="images/pubmed.jpg
                                 "
                            width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1038/s41598-017-09654-8"
       target="_blank"><img src="images/doi.jpg
                                 "
                            width="30" height="16" align="absmiddle"
                            style="padding-left:2px"></a></li></ul></p><h2>Results</h2><p>
The SWISS-MODEL template library (SMTL version 2018-11-14, PDB release 2018-11-09) was searched with 
BLAST (<a href="#blastp">Camacho et al.</a>) and HHBlits (<a href="#hhblits">Remmert et al.</a>) 

 for evolutionary related structures matching the target sequence in Table T1. For details on the template search, see Materials and Methods. Overall 152 templates were found (Table T2).</p><h2>Models</h2><p>The following models were built (see Materials and Methods "Model Building"):</p><table class="borderedTbl" width="100%" border=0 cellspacing=1 cellpadding=5><thead><tr><th><h3>Model #05</h3></th><th>File</th><th>Built with</th><th>Oligo-State</th><th>Ligands</th><th title="Global Model Quality Estimate">GMQE</th><th>QMEAN</th></tr></thead><tbody><tr><td><a href="model/05/05.png" target="_blank"><img height="60" src="model/05/05.png"></a></td><td><a href="model/05/model.pdb" target="_blank">PDB</a></td><td>
      ProMod3 Version 1.2.0.
      </td><td>monomer</td><td><div style="max-width:350px;">None</div></td><td>0.44</td><td>-7.38</td></tr></table><table width="100%"><tr><td><table cellpadding="0" cellspacing="2"><tr><td width="80">QMEAN</td><td>-7.38</td><td><img height="16" width="100" src="model/05/qmean/energy_profile_plots/qmean4_compact.png"></td></tr><tr><td>C&#946;</td><td>-6.27</td><td><img height="16" width="100" src="model/05/qmean/energy_profile_plots/cbeta_compact.png"></td></tr><tr><td>All Atom</td><td>-4.24</td><td><img height="16" width="100" src="model/05/qmean/energy_profile_plots/all_atom_compact.png"></td></tr><tr><td>Solvation</td><td>-0.62</td><td><img height="16" width="100" src="model/05/qmean/energy_profile_plots/solvation_compact.png"></td></tr><tr><td>Torsion</td><td>-6.14</td><td><img height="16" width="100" src="model/05/qmean/energy_profile_plots/torsion_compact.png"></td></tr></table></td><td><span class="pull-left" style="margin-right:4px;"><a href="model/05/qmean/colored_PDB/local_quality_estimate.png" target="_blank"><img width="180" height="120" src="model/05/qmean/colored_PDB/local_quality_estimate_A.png"></a></td><td><a href="model/05/qmean/colored_PDB/reference_set.png" target="_blank"><img  width="180" height="120" src="model/05/qmean/colored_PDB/reference_set.png"></a></td></tr></table><table class="borderedTbl"><thead><tr><th>Template</th><th>Seq Identity</th><th>Oligo-state</th><th>Found by</th><th>Method</th><th>Resolution</th><th>Seq Similarity</th><th>Range</th><th>Coverage</th><th>Description</th></tr></thead><tbody><tr><td><a href="model/05/templates/.pdb" target="_blank" >4ybq.1.A</a></td><td>12.81</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>3.27&Aring;</td><td>0.27</td><td>35 - 480</td><td>0.82</td><td>Solute carrier family 2, facilitated glucose transporter member 5</td></tr></tbody></table> 
  	

    
  The template contained no ligands.
  

<br><pre>Target    MATKTELSPTARESKNAQDMQVDETLIPRKVPSLCSARYGIALVLHFC-NFTTIAQNVIMNITMVAMVNSTSPQSQLNDS<br>4ybq.1.A  ----------------------------------TLVLALATFLAAFGSSFQYGYNVAAVNSPSEFMQQFYYDTYY----<br><br>Target    SEVLPVDSFGGLSKAPKSLPAKAPVYDWSPQI----QGIIFGAVGYGGILTMAPSGYLAGRVGTKRVVGISLFATSFLTL<br>4ybq.1.A  ---------------------DRNKENIESFTLTLLWSLTVSMFPFGGFIGSLMVGFLVNNLGRKGALLFNNIFSILPAI<br><br>Target    CIPLA-TDFGIVLLIVTRIVQGLSQSSILGGQFAIWEKWGPPQERSRLCSIALSGMLLGCFTAILIG--GFISETLGWPF<br>4ybq.1.A  LMGCSKIAKSFEIIIASRLLVGICAGISSNVVPMYLGELAPKNLRGALGVVPQLFITVGILVAQLFGLRSVLASEEGWPI<br><br>Target    VFYIFGGVGCVCCLLWFVVIYDDPVSY--PWISTSEKEYIISSLK---------------QQVGSSKQPLPIKAMLRSLP<br>4ybq.1.A  LLGLTGVPAGLQLLLL-PFFPESPRYLLIQKKNESAAEKALQTLRGWKDVDMEMEEIRKEDEAEKAAGFISVWKLFRMQS<br><br>Target    IWSICLGCFSHQW-----LVSTMVVYIPTYISSVYHVNIRDNGLLSALPFIVAWVIGMVGGYLADFLLTKKFRLITVRKI<br>4ybq.1.A  LRWQLISTIVLMAGQQLSGVNAIYYYADQIYLS-AGVKSNDVQYVTAGTGAVNVFMTMVTVFVVELWGRRNLL-------<br><br>Target    ATILGSLPSSALIVSLPYL----NS-GY--ITATA-LLTLSCGLSTLCQSGIYINVLDIAP-RYSSFLMGASRGFSSIAP<br>4ybq.1.A  --LIGFSTCLTACIVLTVALALQNTISWMPYVSIVCVIVYVIGHAVGPSPIPALFITEIFLQSSRPSAYMIGGSVHWLSN<br><br>Target    VIVPTVSGFLLSQDPEFGWRNVFFLLFAVNLLGLLFYLIFGEADVQEWAKERKLTRL<br>4ybq.1.A  FIVGLIFPFIQVG---LGP-YSFIIFAIICLLTTIYIFM------------------<br><br></pre><br><hr><br><table class="borderedTbl" width="100%" border=0 cellspacing=1 cellpadding=5><thead><tr><th><h3>Model #04</h3></th><th>File</th><th>Built with</th><th>Oligo-State</th><th>Ligands</th><th title="Global Model Quality Estimate">GMQE</th><th>QMEAN</th></tr></thead><tbody><tr><td><a href="model/04/04.png" target="_blank"><img height="60" src="model/04/04.png"></a></td><td><a href="model/04/model.pdb" target="_blank">PDB</a></td><td>
      ProMod3 Version 1.2.0.
      </td><td>monomer</td><td><div style="max-width:350px;">None</div></td><td>0.44</td><td>-9.75</td></tr></table><table width="100%"><tr><td><table cellpadding="0" cellspacing="2"><tr><td width="80">QMEAN</td><td>-9.75</td><td><img height="16" width="100" src="model/04/qmean/energy_profile_plots/qmean4_compact.png"></td></tr><tr><td>C&#946;</td><td>-5.99</td><td><img height="16" width="100" src="model/04/qmean/energy_profile_plots/cbeta_compact.png"></td></tr><tr><td>All Atom</td><td>-4.63</td><td><img height="16" width="100" src="model/04/qmean/energy_profile_plots/all_atom_compact.png"></td></tr><tr><td>Solvation</td><td>-0.71</td><td><img height="16" width="100" src="model/04/qmean/energy_profile_plots/solvation_compact.png"></td></tr><tr><td>Torsion</td><td>-8.62</td><td><img height="16" width="100" src="model/04/qmean/energy_profile_plots/torsion_compact.png"></td></tr></table></td><td><span class="pull-left" style="margin-right:4px;"><a href="model/04/qmean/colored_PDB/local_quality_estimate.png" target="_blank"><img width="180" height="120" src="model/04/qmean/colored_PDB/local_quality_estimate_A.png"></a></td><td><a href="model/04/qmean/colored_PDB/reference_set.png" target="_blank"><img  width="180" height="120" src="model/04/qmean/colored_PDB/reference_set.png"></a></td></tr></table><table class="borderedTbl"><thead><tr><th>Template</th><th>Seq Identity</th><th>Oligo-state</th><th>Found by</th><th>Method</th><th>Resolution</th><th>Seq Similarity</th><th>Range</th><th>Coverage</th><th>Description</th></tr></thead><tbody><tr><td><a href="model/04/templates/.pdb" target="_blank" >4zwc.1.A</a></td><td>11.66</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>2.60&Aring;</td><td>0.25</td><td>34 - 480</td><td>0.81</td><td>Solute carrier family 2, facilitated glucose transporter member 3</td></tr></tbody></table><br><table width="100%" class="borderedTbl"><thead><tr><th>Ligand</th><th>Added to Model</th><th>Description</th></tr></thead><tbody><tr><td>MAL</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">MALTOSE</div></td></tr><tr><td>OLC</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">(2R)-2,3-dihydroxypropyl (9Z)-octadec-9-enoate</div></td></tr><tr><td>OLC</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">(2R)-2,3-dihydroxypropyl (9Z)-octadec-9-enoate</div></td></tr></tbody></table><br><pre>Target    MATKTELSPTARESKNAQDMQVDETLIPRKVPSLCSARYGIALVLHFCNFTTIAQNVIMNITMVAMVNSTSPQSQLNDSS<br>4zwc.1.A  ---------------------------------VTPALIFAITVATIGSFQFGYNTGVINAPEKIIKEFITKT-------<br><br>Target    EVLPVDSFGGLSKAPKSLPAKAPVYDWSPQI--Q----GIIFGAVGYGGILTMAPSGYLAGRVGTKRVVGISLFATSFLT<br>4zwc.1.A  --------------------LTDKGNAPPSEVLLTSLWSLSVAIFSVGGMIGSFSVGLFVNRFGRRNSMLIVNLLAVTGG<br><br>Target    LCIPLA-TDFGIVLLIVTRIVQGLSQSSILGGQFAIWEKWGPPQERSRLCSIALSGMLLGCFTAILIGGF--ISETLGWP<br>4zwc.1.A  CFMGLCKVAKSVEMLILGRLVIGLFCGLCTGFVPMYIGEISPTALRGAFGTLNQLGIVVGILVAQIFGLEFILGSEELWP<br><br>Target    FVFYIFGGVGCVCCLLWFVVIYDDPVS-YPWI-STSEKEYIISSLK---------------QQVGSSKQPLPIKAMLRSL<br>4zwc.1.A  LLLGFTILPAILQSAAL-PFCPESPRFLLINRKEEENAKQILQRLWGTQDVSQDIQEMKDESARMSQEKQVTVLELFRVS<br><br>Target    PIWSICL----GCFSH-QWLVSTMVVYIPTYISSVYHVNIRDNGLLSALPFIVAWVIGMVGGYLADFLLTKKFRLITVRK<br>4zwc.1.A  SYRQPIIISIVLQLSQQLSGINAVFYYSTGIFKD-AGVQE-PI-YATIGAGVVNTIFTVVSLFLVERAGRRTLH------<br><br>Target    IATILGSLPSSALIVSLPY----LN-S--GYITATALLTLSCGLSTL-CQSGIYINVLDIAP-RYSSFLMGASRGFSSIA<br>4zwc.1.A  ---MIGLGGMAFCSTLMTVSLLLKDNYNGMSFVCIGAILVFVAFFEIGPGPIPWFIVAELFSQGPRPAAMAVAGCSNWTS<br><br>Target    PVIVPTVSGFLLSQDPEFGWRNVFFLLFAVNLLGLLFYLIFGEADVQEWAKERKLTRL<br>4zwc.1.A  NFLVGLLFPSAAHY---LG-AYVFIIFTGFLITFLAFTFF------------------<br><br></pre><br><hr><br><table class="borderedTbl" width="100%" border=0 cellspacing=1 cellpadding=5><thead><tr><th><h3>Model #06</h3></th><th>File</th><th>Built with</th><th>Oligo-State</th><th>Ligands</th><th title="Global Model Quality Estimate">GMQE</th><th>QMEAN</th></tr></thead><tbody><tr><td><a href="model/06/06.png" target="_blank"><img height="60" src="model/06/06.png"></a></td><td><a href="model/06/model.pdb" target="_blank">PDB</a></td><td>
      ProMod3 Version 1.2.0.
      </td><td>monomer</td><td><div style="max-width:350px;">None</div></td><td>0.40</td><td>-9.49</td></tr></table><table width="100%"><tr><td><table cellpadding="0" cellspacing="2"><tr><td width="80">QMEAN</td><td>-9.49</td><td><img height="16" width="100" src="model/06/qmean/energy_profile_plots/qmean4_compact.png"></td></tr><tr><td>C&#946;</td><td>-5.57</td><td><img height="16" width="100" src="model/06/qmean/energy_profile_plots/cbeta_compact.png"></td></tr><tr><td>All Atom</td><td>-4.53</td><td><img height="16" width="100" src="model/06/qmean/energy_profile_plots/all_atom_compact.png"></td></tr><tr><td>Solvation</td><td>-0.95</td><td><img height="16" width="100" src="model/06/qmean/energy_profile_plots/solvation_compact.png"></td></tr><tr><td>Torsion</td><td>-8.39</td><td><img height="16" width="100" src="model/06/qmean/energy_profile_plots/torsion_compact.png"></td></tr></table></td><td><span class="pull-left" style="margin-right:4px;"><a href="model/06/qmean/colored_PDB/local_quality_estimate.png" target="_blank"><img width="180" height="120" src="model/06/qmean/colored_PDB/local_quality_estimate_A.png"></a></td><td><a href="model/06/qmean/colored_PDB/reference_set.png" target="_blank"><img  width="180" height="120" src="model/06/qmean/colored_PDB/reference_set.png"></a></td></tr></table><table class="borderedTbl"><thead><tr><th>Template</th><th>Seq Identity</th><th>Oligo-state</th><th>Found by</th><th>Method</th><th>Resolution</th><th>Seq Similarity</th><th>Range</th><th>Coverage</th><th>Description</th></tr></thead><tbody><tr><td><a href="model/06/templates/.pdb" target="_blank" >5c65.1.A</a></td><td>11.91</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>2.65&Aring;</td><td>0.25</td><td>34 - 480</td><td>0.81</td><td>Solute carrier family 2, facilitated glucose transporter member 3</td></tr></tbody></table><br><table width="100%" class="borderedTbl"><thead><tr><th>Ligand</th><th>Added to Model</th><th>Description</th></tr></thead><tbody><tr><td>37X</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">Octyl Glucose Neopentyl Glycol</div></td></tr><tr><td>37X</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">Octyl Glucose Neopentyl Glycol</div></td></tr><tr><td>37X</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">Octyl Glucose Neopentyl Glycol</div></td></tr><tr><td>37X</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">Octyl Glucose Neopentyl Glycol</div></td></tr><tr><td>37X</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">Octyl Glucose Neopentyl Glycol</div></td></tr><tr><td>37X</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">Octyl Glucose Neopentyl Glycol</div></td></tr><tr><td>Y01</td><td>&#10005; -
      
        Binding site not conserved. 
          
        
      
      </td><td><div style="max-width:650px;">CHOLESTEROL HEMISUCCINATE</div></td></tr></tbody></table><br><pre>Target    MATKTELSPTARESKNAQDMQVDETLIPRKVPSLCSARYGIALVLHFCNFTTIAQNVIMNITMVAMVNSTSPQSQLNDSS<br>5c65.1.A  ---------------------------------VTPALIFAITVATIGSFQFGYNTGVINAPEKIIKEFIQK--------<br><br>Target    EVLPVDSFGGLSKAPKSLPAKAPVYDWS--PQIQ----GIIFGAVGYGGILTMAPSGYLAGRVGTKRVVGISLFATSFLT<br>5c65.1.A  -------------------TLTDKGNAPPSEVLLTSLWSLSVAIFSVGGMIGSFSVGLFVNRFGRRNSMLIVNLLAVTGG<br><br>Target    LCIPLAT-DFGIVLLIVTRIVQGLSQSSILGGQFAIWEKWGPPQERSRLCSIALSGMLLGCFTAILIGGFIS--ETLGWP<br>5c65.1.A  CFMGLCKVAKSVEMLILGRLVIGLFCGLCTGFVPMYIGEISPTALRGAFGTLNQLGIVVGILVAQIFGLEFILGSEELWP<br><br>Target    FVFYIFGGVGCVCCLLWFVVIYDDPVSY--PWISTSEKEYIISSL-------------K--QQVGSSKQPLPIKAMLRSL<br>5c65.1.A  LLLGFTILPAILQSAAL-PFCPESPRFLLINRKEEENAKQILQRLWGTQDVSQDIQEMKDESARMSQEKQVTVLELFRVS<br><br>Target    PIWSICLG----CFSHQ-WLVSTMVVYIPTYISSVYHVNIRDNGLLSALPFIVAWVIGMVGGYLADFLLTKKFRLITVRK<br>5c65.1.A  SYRQPIIISIVLQLSQQLSGINAVFYYSTGIFKD-AGVQE-PI-YATIGAGVVNTIFTVVSLFLVERAGRRTLH------<br><br>Target    IATILGSLPSSALIVSLPYL----N--S-GYITATALLTLSCGLSTL-CQSGIYINVLDIAP-RYSSFLMGASRGFSSIA<br>5c65.1.A  ---MIGLGGMAFCSTLMTVSLLLKDNYNGMSFVCIGAILVFVAFFEIGPGPIPWFIVAELFSQGPRPAAMAVAGCSNWTS<br><br>Target    PVIVPTVSGFLLSQDPEFGWRNVFFLLFAVNLLGLLFYLIFGEADVQEWAKERKLTRL<br>5c65.1.A  NFLVGLLFPSAAHY---LG-AYVFIIFTGFLITFLAFTFF------------------<br><br></pre><br><hr><br><h2>Materials and Methods</h2><h2>Template Search</h2><p>Template search with BLAST and HHBlits
 has been performed against the SWISS-MODEL template library (SMTL, last update: 2018-11-14, last included PDB release: 2018-11-09).</p><p>
The target sequence was searched with BLAST against the primary amino acid sequence contained in the SMTL.



</p><p>
An initial HHblits profile has been built using the procedure outlined in (Remmert et al.), followed by 1 iteration of HHblits against NR20. The obtained profile has then be searched against all profiles of the SMTL. A total of 152 templates were found.</p><h2>Template Selection</h2><p>For each identified template, the template's quality has been predicted from features of the target-template alignment.
 The templates with the highest quality have then been selected for model building.</p><h2>Model Building</h2><p>
Models are built based on the target-template alignment using ProMod3. Coordinates which are conserved between the target and the template are copied from the template to the model. Insertions and deletions are remodelled using a fragment library. Side chains are then rebuilt. Finally, the geometry of the resulting model is regularized by using a force field. In case loop modelling with ProMod3 fails, an alternative model is built with PROMOD-II (<a href="#promodII">Guex et al.</a>).
</p><h2>Model Quality Estimation</h2><p>
The global and per-residue model quality has been assessed using the QMEAN scoring function (<a href="#qmean">Benkert et al.</a>) . For improved performance, weights of the individual QMEAN terms have been trained specifically for SWISS-MODEL. 
</p><h2>Ligand Modelling</h2><p>
Ligands present in the template structure are transferred by homology to the model when the following criteria are met: (a) The ligands are annotated as biologically relevant in the template library, (b) the ligand is in contact with the model, (c) the ligand is not clashing with the protein, (d) the residues in contact with the ligand are conserved between the target and the template. If any of these four criteria is not satisfied, a certain ligand will not be included in the model. The model summary includes information on why and which ligand has not been included.</p><h2>Oligomeric State Conservation</h2><p>
The quaternary structure annotation of the template is used to model the target sequence in its oligomeric form. The method (<a href="#qsqe">Bertoni et al.</a>) is based on a supervised machine learning algorithm, Support Vector Machines (SVM), which combines interface conservation, structural clustering, and other template features to provide a quaternary structure quality estimate (QSQE). The QSQE score is a number between 0 and 1, reflecting the expected accuracy of the interchain contacts for a model built based a given alignment and template. Higher numbers indicate higher reliability. This complements the GMQE score which estimates the accuracy of the tertiary structure of the resulting model.</p><h2>References</h2><p><ul><li><a name="blastp"></a><b>BLAST</b><br>
        Camacho, C., Coulouris, G., Avagyan, V., Ma, N., Papadopoulos, J.,
        Bealer, K., Madden, T.L. BLAST+: architecture and applications. BMC
        Bioinformatics 10, 421-430 (2009).
        <a target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/20003500"><img width="17" align="absmiddle" height="16"
               src="images/pubmed.jpg
                    "></a><a href="https://dx.doi.org/10.1186/1471-2105-10-421"
           target="_blank"><img src="images/doi.jpg
                                     " width="30"
                                height="16" align="absmiddle"
                                style="padding-left:2px"></a></li><li><a name="hhblits"></a><b>HHblits</b><br>
        Remmert, M., Biegert, A., Hauser, A., S&ouml;ding, J. HHblits:
        lightning-fast iterative protein sequence searching by HMM-HMM
        alignment. Nat Methods 9, 173-175 (2012).
        <a target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/22198341"><img width="17" align="absmiddle" height="16"
               src="images/pubmed.jpg
                    "></a><a href="https://dx.doi.org/10.1038/nmeth.1818"
           target="_blank"><img src="images/doi.jpg
                                     " width="30"
                                height="16" align="absmiddle"
                                style="padding-left:2px"></a></li></ul></p><h2>Table T1:</h2><p>Primary amino acid sequence for which templates were searched and models were built.</p><div class="sequence">MATKTELSPTARESKNAQDMQVDETLIPRKVPSLCSARYGIALVLHFCNFTTIAQNVIMNITMVAMVNSTSPQSQLNDSSEVLPVDSFGGLSKAPKSLPA<br>KAPVYDWSPQIQGIIFGAVGYGGILTMAPSGYLAGRVGTKRVVGISLFATSFLTLCIPLATDFGIVLLIVTRIVQGLSQSSILGGQFAIWEKWGPPQERS<br>RLCSIALSGMLLGCFTAILIGGFISETLGWPFVFYIFGGVGCVCCLLWFVVIYDDPVSYPWISTSEKEYIISSLKQQVGSSKQPLPIKAMLRSLPIWSIC<br>LGCFSHQWLVSTMVVYIPTYISSVYHVNIRDNGLLSALPFIVAWVIGMVGGYLADFLLTKKFRLITVRKIATILGSLPSSALIVSLPYLNSGYITATALL<br>TLSCGLSTLCQSGIYINVLDIAPRYSSFLMGASRGFSSIAPVIVPTVSGFLLSQDPEFGWRNVFFLLFAVNLLGLLFYLIFGEADVQEWAKERKLTRL<br></div><h2>Table T2:</h2><table class="borderedTbl"><thead><tr><th>Template</th><th>Seq Identity</th><th>Oligo-state</th><th>QSQE</th><th>Found by</th><th>Method</th><th>Resolution</th><th>Seq Similarity</th><th>Coverage</th><th>Description</th></tr></thead><tbody><tr><td>4yb9.1.A</td><td>13.05</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.20&Aring;</td><td>0.26</td><td>0.82</td><td>Solute carrier family 2, facilitated glucose transporter member 5</td></tr><tr><td>1pw4.1.A</td><td>13.75</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.30&Aring;</td><td>0.27</td><td>0.80</td><td>Glycerol-3-phosphate transporter</td></tr><tr><td>3wdo.1.A</td><td>14.29</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.15&Aring;</td><td>0.27</td><td>0.76</td><td>MFS Transporter</td></tr><tr><td>4zw9.1.A</td><td>11.66</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>1.50&Aring;</td><td>0.25</td><td>0.81</td><td>Solute carrier family 2, facilitated glucose transporter member 3</td></tr><tr><td>4zwc.1.A</td><td>11.66</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.60&Aring;</td><td>0.25</td><td>0.81</td><td>Solute carrier family 2, facilitated glucose transporter member 3</td></tr><tr><td>5eqg.1.A</td><td>11.50</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.90&Aring;</td><td>0.26</td><td>0.80</td><td>Solute carrier family 2, facilitated glucose transporter member 1</td></tr><tr><td>4ybq.1.A</td><td>12.81</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.27&Aring;</td><td>0.27</td><td>0.82</td><td>Solute carrier family 2, facilitated glucose transporter member 5</td></tr><tr><td>6gv1.1.A</td><td>15.88</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.40&Aring;</td><td>0.27</td><td>0.72</td><td>Major Facilitator Superfamily multidrug/H+ antiporter MdfA from E.coli</td></tr><tr><td>4zow.1.A</td><td>15.88</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.45&Aring;</td><td>0.27</td><td>0.72</td><td>Multidrug transporter MdfA</td></tr><tr><td>4gby.1.A</td><td>13.05</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.81&Aring;</td><td>0.27</td><td>0.82</td><td>D-xylose-proton symporter</td></tr><tr><td>6c9w.1.A</td><td>15.17</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.00&Aring;</td><td>0.27</td><td>0.71</td><td>Lactose permease</td></tr><tr><td>4lds.1.A</td><td>15.54</td><td>homo-dimer</td><td>0.01</td><td>HHblits</td><td>X-ray</td><td>3.20&Aring;</td><td>0.28</td><td>0.80</td><td>Bicyclomycin resistance protein TcaB</td></tr><tr><td>4lds.1.B</td><td>15.54</td><td>homo-dimer</td><td>0.01</td><td>HHblits</td><td>X-ray</td><td>3.20&Aring;</td><td>0.28</td><td>0.80</td><td>Bicyclomycin resistance protein TcaB</td></tr><tr><td>4w6v.1.A</td><td>15.24</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.02&Aring;</td><td>0.26</td><td>0.75</td><td>Di-/tripeptide transporter</td></tr><tr><td>6exs.1.A</td><td>12.47</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.50&Aring;</td><td>0.25</td><td>0.74</td><td>Peptide ABC transporter permease</td></tr><tr><td>4oaa.1.A</td><td>15.17</td><td>homo-dimer</td><td>0.10</td><td>HHblits</td><td>X-ray</td><td>3.50&Aring;</td><td>0.27</td><td>0.71</td><td>Lactose/galactose transporter</td></tr><tr><td>4oaa.1.B</td><td>15.17</td><td>homo-dimer</td><td>0.10</td><td>HHblits</td><td>X-ray</td><td>3.50&Aring;</td><td>0.27</td><td>0.71</td><td>Lactose/galactose transporter</td></tr><tr><td>4zyr.1.A</td><td>15.17</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.31&Aring;</td><td>0.27</td><td>0.71</td><td>Lactose permease</td></tr><tr><td>4ja4.1.A</td><td>13.66</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>4.20&Aring;</td><td>0.27</td><td>0.82</td><td>D-xylose-proton symporter</td></tr><tr><td>3o7q.1.A</td><td>13.19</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.14&Aring;</td><td>0.26</td><td>0.76</td><td>L-fucose-proton symporter</td></tr><tr><td>4qiq.1.A</td><td>13.73</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.51&Aring;</td><td>0.27</td><td>0.82</td><td>D-xylose-proton symporter</td></tr><tr><td>4zyr.2.A</td><td>15.17</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.31&Aring;</td><td>0.27</td><td>0.71</td><td>Lactose permease</td></tr><tr><td>4ikw.1.A</td><td>14.44</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.00&Aring;</td><td>0.26</td><td>0.74</td><td>Di-tripeptide ABC transporter (Permease)</td></tr><tr><td>4ikv.1.A</td><td>14.44</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>1.90&Aring;</td><td>0.26</td><td>0.74</td><td>Di-tripeptide ABC transporter (Permease)</td></tr><tr><td>4ja3.1.A</td><td>13.66</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.80&Aring;</td><td>0.27</td><td>0.82</td><td>D-xylose-proton symporter</td></tr><tr><td>4tpg.2.A</td><td>12.90</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.91&Aring;</td><td>0.26</td><td>0.75</td><td>Proton:oligopeptide symporter POT family</td></tr><tr><td>3o7p.1.A</td><td>13.00</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.20&Aring;</td><td>0.26</td><td>0.76</td><td>L-fucose-proton symporter</td></tr><tr><td>4ikx.1.A</td><td>14.13</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.30&Aring;</td><td>0.25</td><td>0.74</td><td>Di-tripeptide ABC transporter (Permease)</td></tr><tr><td>4iky.1.A</td><td>14.13</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.10&Aring;</td><td>0.25</td><td>0.74</td><td>Di-tripeptide ABC transporter (Permease)</td></tr><tr><td>4iu8.2.A</td><td>11.53</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.11&Aring;</td><td>0.25</td><td>0.75</td><td>Nitrite extrusion protein 2</td></tr><tr><td>4u4t.1.A</td><td>9.89</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.40&Aring;</td><td>0.25</td><td>0.75</td><td>Nitrate/nitrite transporter NarK</td></tr><tr><td>4u4w.1.A</td><td>9.89</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.40&Aring;</td><td>0.25</td><td>0.75</td><td>Nitrate/nitrite transporter NarK</td></tr><tr><td>5oxq.1.A</td><td>12.26</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.19&Aring;</td><td>0.25</td><td>0.74</td><td>Di-or tripeptide:H+ symporter</td></tr><tr><td>4tpj.1.A</td><td>12.90</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.20&Aring;</td><td>0.26</td><td>0.75</td><td>Proton:oligopeptide symporter POT family</td></tr><tr><td>5d58.1.A</td><td>12.16</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.40&Aring;</td><td>0.25</td><td>0.74</td><td>Di-or tripeptide:H+ symporter</td></tr><tr><td>4lep.1.A</td><td>12.90</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.20&Aring;</td><td>0.26</td><td>0.75</td><td>Proton:oligopeptide symporter POT family</td></tr><tr><td>5c65.1.A</td><td>11.91</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.65&Aring;</td><td>0.25</td><td>0.81</td><td>Solute carrier family 2, facilitated glucose transporter member 3</td></tr><tr><td>4pyp.1.A</td><td>11.50</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.17&Aring;</td><td>0.26</td><td>0.80</td><td>Solute carrier family 2, facilitated glucose transporter member 1</td></tr><tr><td>4iu8.1.A</td><td>11.53</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.11&Aring;</td><td>0.25</td><td>0.75</td><td>Nitrite extrusion protein 2</td></tr><tr><td>6fmy.1.A</td><td>12.16</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.70&Aring;</td><td>0.25</td><td>0.74</td><td>Di-or tripeptide:H+ symporter</td></tr><tr><td>4j05.1.A</td><td>12.72</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.90&Aring;</td><td>0.26</td><td>0.81</td><td>Phosphate transporter</td></tr><tr><td>4xnj.1.A</td><td>12.16</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.30&Aring;</td><td>0.25</td><td>0.74</td><td>Di-or tripeptide:H+ symporter</td></tr><tr><td>4jr9.1.A</td><td>9.92</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.60&Aring;</td><td>0.25</td><td>0.75</td><td>Nitrite extrusion protein 1</td></tr><tr><td>5a2n.1.B</td><td>10.96</td><td>homo-dimer</td><td>0.08</td><td>HHblits</td><td>X-ray</td><td>3.70&Aring;</td><td>0.25</td><td>0.75</td><td>PROTEIN NRT1/ PTR FAMILY 6.3</td></tr><tr><td>4aps.1.A</td><td>12.20</td><td>homo-dimer</td><td>0.05</td><td>HHblits</td><td>X-ray</td><td>3.30&Aring;</td><td>0.25</td><td>0.74</td><td>DI-OR TRIPEPTIDE H+ SYMPORTER</td></tr><tr><td>5a2n.1.A</td><td>10.96</td><td>homo-dimer</td><td>0.08</td><td>HHblits</td><td>X-ray</td><td>3.70&Aring;</td><td>0.25</td><td>0.75</td><td>PROTEIN NRT1/ PTR FAMILY 6.3</td></tr><tr><td>4d2b.1.A</td><td>12.20</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.35&Aring;</td><td>0.25</td><td>0.74</td><td>DI-OR TRIPEPTIDE\:H+ SYMPORTER</td></tr><tr><td>4iu9.2.A</td><td>11.53</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>3.01&Aring;</td><td>0.25</td><td>0.75</td><td>Nitrite extrusion protein 2</td></tr><tr><td>4d2c.1.A</td><td>12.20</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.47&Aring;</td><td>0.25</td><td>0.74</td><td>Di-or tripeptide:H+ symporter</td></tr><tr><td>4xni.1.A</td><td>12.16</td><td>monomer</td><td></td><td>HHblits</td><td>X-ray</td><td>2.80&Aring;</td><td>0.25</td><td>0.74</td><td>Di-or tripeptide:H+ symporter</td></tr></tbody></table><br>The table above shows the top 50 filtered templates. A further 76 templates were found which were considered to be less suitable for modelling than the filtered list.<br>
4m64.2.A, 5mmt.1.A, 3o7q.1.A, 6ei3.1.A, 4m64.3.A, 5gxb.1.A, 4ikw.1.A, 2v8n.1.A, 4ikv.1.A, 5d58.1.A, 6fmy.1.A, 4m64.4.A, 4m64.1.A, 6exs.1.A, 4zow.1.A, 4ybq.1.A, 4j05.1.A, 6gv1.1.A, 4w6v.1.A, 3wdo.1.A, 4oaa.1.A, 4oaa.1.B, 4ikx.1.A, 4iu8.1.A, 4yb9.1.A, 4uvm.1.A, 6euq.1.A, 4iky.1.A, 2xut.2.A, 2y5y.1.B, 5oxl.1.A, 5a2n.1.B, 4zp0.1.A, 4xni.1.A, 4iu9.2.A, 4q65.1.A, 4ja3.1.A, 4zwc.1.A, 4pyp.1.A, 4d2b.1.A, 1pv6.1.A, 4ja4.1.A, 5ayo.1.A, 4qiq.1.A, 5oxp.1.A, 4lep.1.A, 4gby.1.A, 5a2n.1.A, 6c9w.1.A, 2cfp.1.A, 4d2c.1.A, 3o7p.1.A, 4u4w.1.A, 2y5y.1.A, 2cfq.1.A, 4zyr.1.A, 4jr9.1.A, 1pw4.1.A, 4lds.1.A, 4tpg.2.A, 5oxq.1.A, 4tpj.1.A, 5aym.1.A, 4iu8.2.A, 6btx.1.A, 4xnj.1.A, 4oh3.1.A, 4u4t.1.A, 4oh3.1.B, 4lds.1.B, 4zw9.1.A, 2gfp.1.A, 4aps.1.A, 4zyr.2.A, 5eqg.1.A, 5c65.1.A
 


	</div><div id="sib_footer"><div id="sib_footer_content"><a href="https://www.sib.swiss">Swiss Institute of Bioinformatics</a><a href="mailto:help-swissmodel@unibas.ch" style="margin-left:16px;">Contact Us</a><div id="sib_footer_right" style="float:right"></div></div></div></body></html>
